components/dashboard/src/components/PrebuildLogs.tsx

```diff
@@ -41,6 +48,10 @@ export default function PrebuildLogs(props: PrebuildLogsProps) {
                     setWorkspace(info.workspace);
                     setWorkspaceInstance(info.latestInstance);
                 }
+                if (props.prebuildId) {
```

Comment:
We should always be able to retrieve the prebuild shape. For that we could use a method called `findPrebuildByWorkspaceId(workspaceId)`.

Reply:
Currently the only PrebuildLogs needing a prebuild are CreateWorkspace and Prebuild, and both have this information already and it gets passed into the component as a prop. Do we need to make this extra call?

Reply:
Oh wait StartWorkspace also

Reply:
> Do we need to make this extra call?

It's not an extra call. So far we doing 1 call (getPrebuild) sometimes, and I'm arguing for doing 1 call (findPrebuildByWorkspaceId) always.
If we wanted to optimize we could think about accepting a full `PrebuildInfo` shape to avoid rendering placeholders but we have to do that one call anyway to be sure we're up-to-date and avoid lost updates.

Reply:
Incorporated `findPrebuildByWorkspaceId` and got rid of the `prebuildId` props 👍🏼